DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs

```diff
@@ -140,13 +140,13 @@ public ApplicationController(IIdentityService identityService, IApplicationServi
             var employerEmailTemplatePath = System.Web.Hosting.HostingEnvironment.MapPath(@"~/App_Data/EmployerEmailTemplate.txt");
             var employerEmailTemplateString = File.ReadAllText(employerEmailTemplatePath);
             var emailContents = _emailService.PrepareApplicationEmailContents(submission, certificationTeamEmailTemplateString, employerEmailTemplateString, EmailReceiver.Both);
+            var pdfName = string.Format("14c_Application_Submission_{0}_{1}_{2}.pdf", submission.Employer.LegalName, submission.Employer.PhysicalAddress.State, DateTime.Now.Year);
```

Comment:
@dciceman David, can you provide a sample file name to see how long this is ending up to be and how it looks? Thanks!

Reply:
14c_Application_Submission_My Test Employer_AL_2018.pdf

Reply:
@david if you could please make this: 14c_Application_AL_2018-01-08_My-Test-Employer.pdf 
basically, spaces replaced with hyphen, full date in the format; order changed so when files are stored in folder CPS folks can easily organize/view them. Thanks!

Reply:
@mmurthydol name format updated.

Reply:
Thanks David! Looks great :) All good from my perspective for the PR. I haven't checked anything that Harun/Greg might check like Handlebars - those changes looked straightforward though.